### ISSUE TITLE
Rename "undo & redo" to "back & forward"

### ISSUE THREAD

**Issue by egraether** [CoatiSoftware/CoatiBugTracker#19]

**Comment by pwinter1991** [comment]
Renaming + binding the typical mouse buttons would be great. Undo/Redo just doesn't fit to the functionality of the buttons.

**Comment by egraether** [comment]
Fixed in 0.3.0.59
The change also includes the typical mouse buttons. The default key codes are 8 and 16, but they can be changed in the ApplicationSettings.xml by defining:
```
<config>
...
	<controls>
		<mouse_back_button><!-- INTEGER: mouse button code of back button, default is 8 --></mouse_back_button>
		<mouse_forward_button><!-- INTEGER: mouse button code of forward button, default is 16 --></mouse_forward_button>
	</controls>
...
</config
```
Also you can go back by pressing BACKSPACE now.
